src/libraries/System.Private.CoreLib/src/System/Type.cs

```diff
@@ -184,7 +184,16 @@ public virtual Type[] GetGenericParameterConstraints()
         [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields)]
         public abstract FieldInfo[] GetFields(BindingFlags bindingAttr);
 
-        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
+        [DynamicallyAccessedMembers(
+            DynamicallyAccessedMemberTypes.PublicFields
+            | DynamicallyAccessedMemberTypes.PublicMethods
+            | DynamicallyAccessedMemberTypes.PublicEvents
+            | DynamicallyAccessedMemberTypes.PublicProperties
+            | DynamicallyAccessedMemberTypes.PublicConstructors
+            | DynamicallyAccessedMemberTypes.PublicFields
```

Comment:
Same as above - duplicate `PublicFields`

Reply:
And also below some more...

Reply:
Sigh. Fixed.

I hope the _one user_ of this useless API is going to appreciate the size savings from the restriction on `.All` because we just spent quite a bit of Microsoft money on this.

Reply:
Maybe this is an indicator we are missing `DynamicallyAccessedMemberTypes.AllPublic` enum value

Reply:
> Maybe this is an indicator we are missing DynamicallyAccessedMemberTypes.AllPublic enum value

We can consider that - would that annotation be useful outside these two rarely used APIs?